src/Mod/Sketcher/Gui/CommandSketcherBSpline.cpp

```diff
@@ -359,6 +359,9 @@ bool isCommandNeedingBSplineKnotActive(Gui::Document* doc)
         }
 
         auto* Obj = static_cast<Sketcher::SketchObject*>(sel[0].getObject());
+        if (!Obj) {
+            return false;
+        }
```

Comment:
This check is not necessary I think. It does not play a role fixing the bug and I have not seen selection objects getObject being tested before. Although I may be wrong about this.
Anyway an additional check like this does not hurt and is good practice I guess.